libs/domain/picking/src/components/picking-in-progress/stories/demo.stories.ts

```diff
@@ -0,0 +1,28 @@
+import { Meta, Story } from '@storybook/web-components';
+import { html, TemplateResult } from 'lit';
+import { storybookPrefix } from '../../../../.constants';
+
+export default {
+  title: `${storybookPrefix}/Picking In Progress Modal`,
+} as Meta;
+
+const Template: Story<unknown> = (): TemplateResult => {
+  return html`<script>
+      (() => {
+        const modalBtn = document.querySelector('#modalBtn');
+
+        modalBtn?.addEventListener('click', () => {
+          const modal = document
+            .querySelector('oryx-picking-in-progress-modal')
+            ?.renderRoot.querySelector('oryx-modal');
+          modal?.open();
+        });
+      })();
+    </script>
+
+    <button id="modalBtn">Open modal</button>
+
+    <oryx-picking-in-progress-modal></oryx-picking-in-progress-modal> `;
```

Comment:
Addition to this comment, if|when it will be implemented: https://github.com/spryker/oryx/pull/328#discussion_r1196485986

Make it opened by default and drop this toggle button and it will be much more valuable for regression. Chromatic can cut the snapshot, due to modal has absolute positioning, don't forget to add OverlaysDecorator like here: 

https://github.com/spryker/oryx/blob/ac8a68d39b1e18f6b7e52cacaea62622634ccdd2/libs/base/ui/overlays/modal/src/stories/static/close-button.stories.ts#LL8C1-L8C1